docs/HowTo/Get-Started/System-Requirements.md

```diff
@@ -1,64 +1,69 @@
-    synchronized to the chain head. 
 
-## RAM 
+    CPU requirements are highest when syncing to the network and typically reduce after the node is
+    synchronized to the chain head.
 
-Besu requires 4GB of RAM. For public Ethereum networks including Mainnet and testnets such as Ropsten, 
-a minimum of 8GB of RAM is required. 
+## RAM
 
-## Disk Space 
+Besu requires 4GB of RAM. For public Ethereum networks, including MainNet and testnets such as
+Ropsten, Besu requires a minimum of 8GB of RAM.
 
-!!! caution 
-    Do not use pruning in Hyperledger Besu v1.4.0. Pruning has a [known bug](https://github.com/hyperledger/besu/blob/master/CHANGELOG.md#known-issues).
-    
-    If using fast sync in v1.4.0, explicitly disable pruning using [`--pruning-enabled=false`](../../Reference/CLI/CLI-Syntax.md#pruning-enabled).
+## Disk space
 
-Syncing to the Ethereum Mainnet requires 3TB for a full sync. To reduce the requirement, use 
-[pruning](../../Concepts/Pruning.md). With pruning, we recommend approximately 750GB of available disk 
-space. 
+!!! caution
 
-## VM Requirements
+    Do not use pruning in Hyperledger Besu v1.4.0. Pruning has a
+    [known bug](https://github.com/hyperledger/besu/blob/master/CHANGELOG.md#known-issues).
 
-You can run Besu on a virtual machine (VM) on a cloud service such as AWS or Azure, or locally
-using a VM manager such as [VirtualBox](https://www.virtualbox.org/).
+    If using fast sync in v1.4.0, explicitly disable pruning using
+    [`--pruning-enabled=false`](../../Reference/CLI/CLI-Syntax.md#pruning-enabled).
 
-If you set up your own VM locally using a VM manager such as [VirtualBox](https://www.virtualbox.org/),
-there are a few considerations:
+Syncing to the Ethereum Mainnet requires 3TB for a full sync. To reduce the requirement, use
+[pruning](../../Concepts/Pruning.md). With pruning, we recommend approximately 750GB of available
+disk space.
 
-* Ensure that Intel Virtualization Technology (VTx) and Virtualization Technology for Directed
-I/O (VT-d) are enabled in BIOS settings.
+## VM requirements
 
-* On Windows, you might need to disable Hyper-V in the Windows Feature list.
+You can run Besu on a virtual machine (VM) on a cloud service, such as AWS or Azure, or locally
+using a VM manager, such as [VirtualBox](https://www.virtualbox.org/).
 
-It is recommended that you create a VM with the following attributes:
+If you set up your own VM locally using a VM manager, there are a few considerations:
 
-* Memory Size: Set to 4096 (recommended)
+* Ensure you enable Intel Virtualization Technology (VTx) and Virtualization Technology for
+  Directed I/O (VT-d) in the BIOS settings.
+* On Windows, you might need to disable Hyper-V in the Windows Feature list.
 
-* Create a virtual hard disk with at least 10 GB; 20 GB is recommended
+We recommend you create a VM with the following attributes:
 
+* Memory Size: Set to 4096 (recommended)
```

Comment:
Could you add the unit? 4096 **MB**
MB is for Mega Bytes, should all be uppercased and with a space between the value and the unit.

Reply:
Done.